coffee-style.css

```diff
@@ -288,4 +288,42 @@ gjjfj
   height: 72px;
 }
 
+/* sectiunea mea : Alex Oniga */
+.commitment-container{
+  width: 1250px;
```

Comment:
aici are trebui un:  width:1110px; daca ne uitam in figma

Reply:
si un height: 520px;

Reply:
iar pentru aliniere ar mai trebui un margin: 0 auto;

Reply:
done!